File: app/src/main/java/uk/co/ourfriendirony/medianotifier/clients/tmdb/tvseason/get/TVSeasonGetExternalIds.java
package uk.co.ourfriendirony.medianotifier.clients.tmdb.tvseason.get;

import com.fasterxml.jackson.annotation.JsonAnyGetter;
import com.fasterxml.jackson.annotation.JsonAnySetter;
import com.fasterxml.jackson.annotation.JsonIgnore;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonPropertyOrder;

import java.util.HashMap;
import java.util.Map;

@JsonInclude(JsonInclude.Include.NON_NULL)
@JsonPropertyOrder({
        "freebase_mid",
        "freebase_id",
        "tvdb_id",
        "tvrage_id"
})
public class TVSeasonGetExternalIds {

    @JsonProperty("freebase_mid")
    private String freebaseMid;
    @JsonProperty("freebase_id")
    private String freebaseId;
    @JsonProperty("tvdb_id")
    private Integer tvdbId;
    @JsonProperty("tvrage_id")
    private Object tvrageId;
    @JsonIgnore
    private Map<String, Object> additionalProperties = new HashMap<String, Object>();

    @JsonProperty("freebase_mid")
    public String getFreebaseMid() {
        return freebaseMid;
    }

    @JsonProperty("freebase_mid")
    public void setFreebaseMid(String freebaseMid) {
        this.freebaseMid = freebaseMid;
    }

    @JsonProperty("freebase_id")
    public String getFreebaseId() {
        return freebaseId;
    }

    @JsonProperty("freebase_id")
    public void setFreebaseId(String freebaseId) {
        this.freebaseId = freebaseId;
    }

    @JsonProperty("tvdb_id")
    public Integer getTvdbId() {
        return tvdbId;
    }

    @JsonProperty("tvdb_id")
    public void setTvdbId(Integer tvdbId) {
        this.tvdbId = tvdbId;
    }

    @JsonProperty("tvrage_id")
    public Object getTvrageId() {
        return tvrageId;
    }

    @JsonProperty("tvrage_id")
    public void setTvrageId(Object tvrageId) {
        this.tvrageId = tvrageId;
    }

    @JsonAnyGetter
    public Map<String, Object> getAdditionalProperties() {
        return this.additionalProperties;
    }

    @JsonAnySetter
    public void setAdditionalProperty(String name, Object value) {
        this.additionalProperties.put(name, value);
    }

}

File: app/src/main/kotlin/uk/co/ourfriendirony/medianotifier/clients/tmdb/tvseason/get/TVSeasonGetExternalIds.kt
package uk.co.ourfriendirony.medianotifier.clients.tmdb.tvseason.get

import com.fasterxml.jackson.annotation.*

@JsonInclude(JsonInclude.Include.NON_NULL)
@JsonIgnoreProperties(ignoreUnknown = true)
@JsonPropertyOrder("freebase_mid", "freebase_id", "tvdb_id", "tvrage_id")
class TVSeasonGetExternalIds {
    @get:JsonProperty("freebase_mid")
    @set:JsonProperty("freebase_mid")
    @JsonProperty("freebase_mid")
    var freebaseMid: String? = null

    @get:JsonProperty("freebase_id")
    @set:JsonProperty("freebase_id")
    @JsonProperty("freebase_id")
    var freebaseId: String? = null

    @get:JsonProperty("tvdb_id")
    @set:JsonProperty("tvdb_id")
    @JsonProperty("tvdb_id")
    var tvdbId: Int? = null

    @get:JsonProperty("tvrage_id")
    @set:JsonProperty("tvrage_id")
    @JsonProperty("tvrage_id")
    var tvrageId: Any? = null
}